src/ROUTES.js

```diff
@@ -40,7 +40,7 @@ export default {
     NEW_GROUP: 'new/group',
     NEW_CHAT: 'new/chat',
     REPORT,
-    REPORT_WITH_ID: 'r/:reportID',
+    REPORT_WITH_ID: 'r/:reportID/:reportActionID?',
```

Comment:
Curious why this is not a new, unique path? I haven't tested the PR yet, but wouldn't that leave `/reportActionID?` at the end of the URL even if the user didn't link to a specific comment?

for example, IOU split has both `IOU_BILL` & `IOU_BILL_WITH_REPORT_ID`. Should we use something like `REPORT_WITH_ID` & `REPORT_WITH_ID_AND_ACTION_ID`